contracts/vaults/STETHVault/RibbonThetaSTETHVault.sol

```diff
@@ -269,16 +271,45 @@ contract RibbonThetaSTETHVault is RibbonVault, RibbonThetaSTETHVaultStorage {
 
         emit InstantWithdraw(msg.sender, amount, currentRound);
 
+        IERC20 steth = IERC20(collateralToken.stETH());
+        uint256 startStethBalance = steth.balanceOf(address(this));
+        bool enoughSTETH = true;
+
+        if (withdrawYieldToken && amount > startStethBalance) {
```

Comment:
if we do need to withdraw, we check if there is enough even with wsteth unwrap and if so we do unwrap